tests/simulate/test_lammps.py

```diff
@@ -246,6 +246,28 @@ def test_minimization(self):
         lmp.operations = emin
         lmp.run(potentials=pot, directory=self.directory)
 
+        # Whole number of iterations and evalulations as float
+        emin.max_iterations = 1000.0
+        emin.options = {"max_evaluations": 10000.0}
+        lmp.operations = emin
+        lmp.run(potentials=pot, directory=self.directory)
+        emin.max_iterations = 1000
+        emin.options = {"max_evaluations": 10000}
```

Comment:
What is the purpose of these lines here? It wasn't clear to me.

Reply:
The order of those tests are: 
1.) whole number iterations and max evaluations as floats
2.) Non whole number of iterations
3.) Non whole number of max evaluations

I thought it was best to reset it after 1 to the default integer state before testing 2 & 3.

Reply:
OK! Maybe add some comments documenting that the first time you use that pattern?

Reply:
I added comments whenever I reset something. I think that should help the readability!